packages/ckeditor5-engine/docs/framework/guides/deep-dive/custom-element-conversion.md

```diff
@@ -298,7 +363,12 @@ class InfoBox {
 			.add( dispatcher => dispatcher.on( 'insert:infoBox', editingDowncastConverter ) );
 		editor.conversion.for( 'dataDowncast' )
 			.add( dispatcher => dispatcher.on( 'insert:infoBox', dataDowncastConverter ) );
-	}
+
+        // Model to view position mapper is needed since the model <infoBox> content needs to end up in the inner
+        // <div class="info-box-content">.
+        editor.editing.mapper.on( 'modelToViewPosition', createModelToViewPositionMapper( editor.editing.view ) );
+        editor.data.mapper.on( 'modelToViewPosition', createModelToViewPositionMapper( editor.editing.view ) );
```

Comment:
Passing `editor.editing.view`? I didn't check how it's used there but it's smelly.

Reply:
Yeah, I was talking with you about it some time ago (while refactoring the to-do lists conversion). It's used there only to be able to create a view range.

Reply:
Do we have a ticket for that? If yes, could you reference it from #7316? If not, please open it.

Reply:
#9466